MetaMask/metamask-extension
# feat: add Lisk network and icon

## **Description**

- Add Lisk mainnet and testnet network data and icon support. 
- Add Lisk mainnet to list of 'featured RPCs', please let me know if there is a special process for this and this change is not allowed.

## **Related issues**

Fixes: N/A

## **Manual testing steps**

- Add Lisk mainnet:
  - Click on the network selection button on the top left.
  - Click on 'Add a network'.
  - Select Lisk from list of proposed networks and add it.
- - Add Lisk Sepolia testnet:
  - Go to [Lisk Sepolia Explorer](https://sepolia-blockscout.lisk.com/)
  - Scroll down to bottom and click the Metamask icon to add network
- Open Metamask extension to check the network icon

## **Screenshots/Recordings**




### **Before**

<img width="770" alt="Screenshot 2024-06-05 at 16 05 28" src="https://github.com/MetaMask/metamask-extension/assets/16502243/a53898a2-574b-4991-84f0-47a8f88cedfe">
<img width="357" alt="Screenshot 2024-06-05 at 16 10 11" src="https://github.com/MetaMask/metamask-extension/assets/16502243/bbae470e-dfb1-4806-b2da-1e9c72d5fb3d">

### **After**

<img width="757" alt="Screenshot 2024-06-05 at 16 13 04" src="https://github.com/MetaMask/metamask-extension/assets/16502243/fc34df15-24db-4d36-9526-ed1b841668b8">
<img width="344" alt="Screenshot 2024-06-05 at 16 12 25" src="https://github.com/MetaMask/metamask-extension/assets/16502243/78985171-e683-4b5b-85fb-d84204923b76">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
